docs/source/howto/plugin_codes.rst

```diff
@@ -454,6 +453,41 @@ Congratulations - you can now write plugins for external simulation codes and us
 
 If you still have time left, consider going through the optional exercise below.
 
+.. _how-to:plugin-codes:importers:
+
+Writing importers for existing computations
+===========================================
+
+.. versionadded:: 2.0
+
+New users to your plugin may often have completed many previous computations without the use of AiiDA, which they wish to import into AiiDA.
+In these cases, it is possible to write an importer for their inputs/outputs, which generates the provenance nodes for the corresponding |CalcJob|.
+
+The importer must be written as a subclass of :class:`~aiida.engine.processes.calcjobs.importer.CalcJobImporter`,
+for an example see :class:`aiida.calculations.importers.arithmetic.add.ArithmeticAddCalculationImporter`.
+
+To associate the importer with the |CalcJob| class, the importer must be registered with the entry point ``aiida.calculations.importers``.
+
+.. code-block:: toml
+
+    [project.entry-points."aiida.calculations.importers"]
+    "core.arithmetic.add" = "aiida.calculations.importers.arithmetic.add:ArithmeticAddCalculationImporter"
+
```

Comment:
```suggestion
.. note::

    Note that the entry point name can be any valid entry point name.
    If the importer plugin is provided by the same package as the corresponding |CalcJob| plugin, it is recommended that the entry point name of the importer and |CalcJob| plugin are the same.
    This will allow the :meth:`~aiida.engine.processes.calcjobs.calcjob.CalcJob:get_importer` method to automatically fetch the associated importer.
    If the entry point names differ, the entry point name of the desired importer implementation needs to be passed to :meth:`~aiida.engine.processes.calcjobs.calcjob.CalcJob:get_importer` as an argument.
```